packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/PreserveStaticIPsDetailsItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useModal } from 'src/modules/Providers/modals';
+import { DetailsItem } from 'src/modules/Providers/utils';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { Label } from '@patternfly/react-core';
+
+import { PlanDetailsItemProps } from '../../DetailsSection';
+import { EditPlanPreserveStaticIPs } from '../modals/EditPlanPreserveStaticIPs';
+
+export const PreserveStaticIPsDetailsItem: React.FC<PlanDetailsItemProps> = ({
+  resource,
+  canPatch,
+  helpContent,
+  destinationProvider,
+}) => {
+  const { t } = useForkliftTranslation();
+  const { showModal } = useModal();
+
+  const defaultHelpContent = t(
+    `Preserve the static IPs that the VM runs with in vSphere provider's environment.`,
```

Comment:
@liranr23 hil
```
Preserve the static IPs of windows virtual machines migrating from vSphere cluster.
```
does that text makes sense ?

Reply:
> @liranr23 hi
> 
> ```
> Preserve the static IPs of windows virtual machines migrating from vSphere cluster.
> ```
> 
> does that text makes sense ?

```
Preserve the static IPs of Windows virtual machines migrating from vSphere cluster.
```

Reply:
@liranr23 
Based on Arik's input (offline), the text will be changed in a follow up to:

`Preserve the static IPs of VMs with Windows guest operating system from vSphere. `